File: Java Barcode Scanner ME/app/src/main/java/de/dimedis/mobileentry/backend/response/ResponseError.java
package de.dimedis.mobileentry.backend.response;

import com.google.gson.annotations.SerializedName;

public class ResponseError {
    public String code;
    public String message;
    public String timestamp;
    @SerializedName("log_token")
    public String logToken;

    @Override
    public String toString() {
        return "ResponseError{" +
                "code='" + code + '\'' +
                ", message='" + message + '\'' +
                ", timestamp='" + timestamp + '\'' +
                ", logToken='" + logToken + '\'' +
                '}';
    }
}

File: Kotlin Migration/NTMobileEntry/app/src/main/java/de/dimedis/mobileentry/backend/response/ResponseError.kt
package de.dimedis.mobileentry.backend.response

import com.google.gson.annotations.SerializedName

class ResponseError {
    var code: String? = null
    var message: String? = null
    var timestamp: String? = null

    @SerializedName("log_token")
    var logToken: String? = null
    override fun toString(): String {
        return "ResponseError{" +
                "code='" + code + '\'' +
                ", message='" + message + '\'' +
                ", timestamp='" + timestamp + '\'' +
                ", logToken='" + logToken + '\'' +
                '}'
    }
}